### PR TITLE
Front end send 5 USD Coin to platform

### PR DESCRIPTION
**Pull request by zhoujc999**

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -23,6 +25,273 @@ const App = () => {
...
+        params: [
+          {
+            from: username,
+            to: tokenContractAddress,
+            // TODO: Method is based on token contract ABI. Hardcoded platform address
+            // Value use string to prevent big num problems
+            data: await tokenContract.methods
+              .transfer(
+                "0x49d0739EB001FF73b394a5A2054694A650dC9cec",
+                toBN(toWei("5", "mwei"))
```
Comment (Jaxonwht) [inline client/src/App.tsx:282]: Can you like take this as an argument? Also send shao yi xie? 5 too much.

**Review by Jaxonwht — CHANGES_REQUESTED** [review]

File: client/src/App.tsx [hunk 2]
```diff
@@ -23,6 +25,273 @@ const App = () => {
...
+        type: "event",
+      },
+    ];
+
+    const tokenContract = new web3.eth.Contract(
+      tokenContractABI,
+      tokenContractAddress
+    );
+
+    await window.ethereum
```
Comment (Jaxonwht) [inline client/src/App.tsx:270]: await 之后接个屁的then

File: client/src/App.tsx (hunk 1 above)
Comment (zhoujc999) [reply client/src/App.tsx:282]: @Jaxonwht 咋搞

Comment (Jaxonwht) [reply client/src/App.tsx:282]: Let me push dead. quick review my pr

Comment (Jaxonwht) [reply client/src/App.tsx:282]: ..

**Review by Jaxonwht — APPROVED** [review]

File: client/src/TokenSender.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        value={amount}
+        onChange={handleAmountInputChanged}
+        type="number"
+        placeholder="Amount of tokens to send"
+      ></input>
+      <input
+        value={fromUser}
+        onChange={handleUserInputChanged}
+        placeholder={"Address to send from"}
+      ></input>
```
Comment (zhoujc999) [inline client/src/TokenSender.tsx:74]: @Jaxonwht 为啥不直接用signed in username？

**Pull request merged**